Doc/library/itertools.rst

```diff
@@ -831,7 +831,7 @@ which incur interpreter overhead.
        return next(g, True) and not next(g, False)
 
    def quantify(iterable, pred=bool):
-       "Count how many times the predicate is true"
+       "Count how many times the predicate is True"
```

Comment:
Ditto; this seems to me to be pedantic churn only. I'd drop this from the PR.

Reply:
No, this is incorrect as well. I also don't see why you're commenting like that when @rhettinger already agreed and asked me to do this.

Reply:
Raymond is one of many core devs. It is not given that changes encouraged by one core dev automatically will be encouraged by another core dev.

Reply:
@erlend-aasland You're a new core-dev and could try to learn from people who have been around for while.

We (collectively) have worked on issues like this before. In general, we've tried to be consistent about using uppercase `True` for the actual boolean object and lowercase `true` for a "truthy" object, one where `bool(obj)` returns `True`.  

In the case of `quantify`, it is not "pedantic" at all.  The function actually sums Trues and Falses which are equivalent to 1 and 0.  A truthy value would not suffice.